server/belga/io/feed_parsers/belga_anp_newsml_1_2.py

```diff
@@ -70,7 +71,7 @@ def parse_newsitem(self, item, newsitem_el):
             ):
                 subject_list = [
                     {"name": i, "qcode": i, "scheme": "original-metadata"}
-                    for i in set(subject["name"].split(";"))
+                    for i in set(re.split('[- ;]', subject["name"]))
```

Comment:
are you sure about the space? how would they send multi word keywords?

Reply:
we can add pipe symbol `re.split('[-|;]'` for better surety.